### PR TITLE
feat: replaced no longer exposed messages and bytes metrics by newly added replacement metrics

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
feat: replaced no longer exposed messages and bytes metrics by newly added replacement metrics

### CODE REVIEW

**Comment by wtrocki** [comment]
ping @pb82 for technical review

File: docs/kafka/metrics-monitoring/README.adoc [hunk 1]
```diff
@@ -90,11 +90,11 @@ Broker metrics::
 Topic metrics::
 +
 --
-* `kafka_server_brokertopicmetrics_bytes_in_total`: Number of incoming bytes to topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`: Number of incoming bytes to topics in the instance in the last five minutes
```
Comment (JameelB) [inline docs/kafka/metrics-monitoring/README.adoc:93]:
```suggestion
* `kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`: Rate of incoming bytes to a topic in the instance in the last five minutes
```

File: docs/kafka/metrics-monitoring/README.adoc [hunk 2]
```diff
@@ -90,11 +90,11 @@ Broker metrics::
...
 
-* `kafka_server_brokertopicmetrics_bytes_out_total`: Number of outgoing bytes from topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_out_total:rate5m`: Number of outgoing bytes from topics in the instance in the last five minutes
```
Comment (JameelB) [inline docs/kafka/metrics-monitoring/README.adoc:95]:
```suggestion
* `kafka_topic:kafka_server_brokertopicmetrics_bytes_out_total:rate5m`: Rate of outgoing bytes from a topic in the instance in the last five minutes
```

File: docs/kafka/metrics-monitoring/README.adoc [hunk 3]
```diff
@@ -90,11 +90,11 @@ Broker metrics::
...
 
-* `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_messages_in_total:rate5m`: Number of messages per second received by one or more topics in the instance in the last five minutes
```
Comment (JameelB) [inline docs/kafka/metrics-monitoring/README.adoc:97]:
```suggestion
* `kafka_topic:kafka_server_brokertopicmetrics_messages_in_total:rate5m`: Rate of messages per second received by a topic in the instance in the last five minutes
```

**Comment by pawelpaszki** [comment]
@JameelB - can you take another look please?

**Review by JameelB — APPROVED** [review]
Thanks, that look good to me :+1:

File: docs/kafka/metrics-monitoring/README.adoc [hunk 4]
```diff
@@ -90,11 +90,11 @@ Broker metrics::
...
 +
 --
-* `kafka_server_brokertopicmetrics_bytes_in_total`: Number of incoming bytes to topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_in_total:rate5m`: Rate of incoming bytes to a topic in the instance in the last five minutes
 
-* `kafka_server_brokertopicmetrics_bytes_out_total`: Number of outgoing bytes from topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_bytes_out_total:rate5m`: Rate of outgoing bytes from a topic in the instance in the last five minutes
 
-* `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance
+* `kafka_topic:kafka_server_brokertopicmetrics_messages_in_total:rate5m`: Rate of messages per second received by a topic in the instance in the last five minutes
```
Comment (JameelB) [inline docs/kafka/metrics-monitoring/README.adoc:98]: After recent discussions it's been agreed on that we won't be removing the old metrics. Let's add these new ones to the end of the list then

Comment (pawelpaszki) [reply docs/kafka/metrics-monitoring/README.adoc]: thanks @JameelB - I have updated my PR

**Review by JameelB — APPROVED** [review]
changes looks good to me :+1: 

@sterobin can you have a look when you get the chance please?

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 5]
```diff
@@ -108,6 +108,12 @@ Topic metrics::
 * `kafka_server_brokertopicmetrics_messages_in_total`: Number of messages per second received by one or more topics in the instance.
 
 * `kafka_topic:kafka_log_log_size:sum`: Log size of each topic and replica, in bytes, across all brokers in the cluster.
```
Comment (sterobin) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:110]: @pawelpaszki So sorry for the delay! I somehow missed the ping :(  You and I already worked through some questions in chat, so my only suggestion is to add periods to the end of each one that you added for consistency, and suggest to move the log size topic metric to the end, to keep the related metrics together.

**Review by sterobin — APPROVED** [review]

**Pull request merged**